### PR TITLE
Updated test names

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

**Review by edward-astapov — APPROVED** [review]

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ test('Check member names and roles in "Our team" block from the "About Us" page
 	await expect(allMemberNames).toHaveText(testDataNames);
 });
 
-test('Check LinkedIn redirects by buttons in "Our team" block from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+test('Check LinkedIn redirects by buttons in "Our team" container from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
 	const ourTeamExperts = driver.getByTestId(AboutUs.OurTeam);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:117]: mb better Check redirects by LinkedIn buttons ... ?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 2]
```diff
@@ -136,7 +136,7 @@ test('Check LinkedIn redirects by buttons in "Our team" block from the "About Us
 	}
 });
 
-test('Check Blog link redirects by buttons in "Our team" block from the "About Us" page @Regression @AboutUs @TSWEB-1022 @TSWEB-1061', async () => {
+test('Check Blog link redirects by buttons in "Our team" container from the "About Us" page @Regression @AboutUs @TSWEB-1022 @TSWEB-1061', async () => {
 	const ourTeamContainer = driver.getByTestId(AboutUs.OurTeam);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:140]: same

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 3]
```diff
@@ -164,14 +164,14 @@ test('Check section titles in "What makes us special" container from the "About
 	await expect(allSectionTitles).toHaveText(testData);
 });
 
-test('Check "Learn more about how we work" button from the "Our partners" block on the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+test('Check "Learn more about how we work" button from the "Our partners" container on the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
 	const ourPartnersBlock = driver.getByTestId(AboutUs.OurPartners);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:168]: mb 
Check "Learn more about how we work" button **in** the "Our partners" container **from** the "About Us" page
?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 4]
```diff
@@ -210,7 +210,7 @@ test('Check redirect by "Clutch Review" button in "Shoutout from our partners" c
 	}
 });
 
-test('Check photo carousel from the "Our people" block from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+test('Check photo carousel from the "Our people" container from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
 	const ourPeopleContainer = driver.getByTestId(AboutUs.OurPeople);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:214]: mb Check photo carousel **in** the "...

File: ui_tests_playwright/automation/test/ui/MainSite/company/HowWeWork.test.ts [hunk 5]
```diff
@@ -14,14 +14,14 @@ test.beforeEach(async () => {
 	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.HowWeWork));
 });
 
-test("Check the header from the 'How we work' block @Regression @HowWeWork", async () => {
+test('Check the header and breadcrumbs from the "How we work" page @Regression @HowWeWork', async () => {
 	const info = driver.getByTestId(HowWeWork.Info);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/HowWeWork.test.ts:18]: mb
Check Info container from the .... ?

File: ui_tests_playwright/automation/test/ui/MainSite/industries/Healthcare/HealthcareContent.test.ts [hunk 6]
```diff
@@ -94,7 +94,7 @@ test("Check section titles, image and CTA button in 'Beats Screening Module by T
 	);
 });
 
-test("Check section numbers and section titles in 'Patient-Centered Strategy' container from the 'Healthcare' block @Regression @Healthcare @TSWEB-955", async () => {
+test('Check section numbers and section titles in "Patient-Centered Strategy" container from the "Healthcare" page @Regression @Healthcare @TSWEB-955', async () => {
 	const patientCenteredStrategyContainer = driver.getByTestId(Healthcare.PatientCenteredStrategy);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/industries/Healthcare/HealthcareContent.test.ts:98]: mb need to remove second section, just 'Check section numbers and titles ... ' ?

File: ui_tests_playwright/automation/test/ui/MainSite/industries/RenewableEnergy/RenewableEnergyContent.test.ts [hunk 7]
```diff
@@ -12,7 +12,7 @@ test.beforeEach(async () => {
 	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.RenewableEnergy));
 });
 
-test("Check the header from the 'Renewable Energy' block @Regression @RenewableEnergy @TSWEB-957", async () => {
+test('Check the header from the "Renewable Energy" page @Regression @RenewableEnergy @TSWEB-957', async () => {
 	const info = driver.getByTestId(RenewableEnergy.Info);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/industries/RenewableEnergy/RenewableEnergyContent.test.ts:16]: same for Info container ?

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

**Pull request merged**